### ISSUE TITLE
Doesn't seem to work any more

### ISSUE THREAD

**Issue by antony** [DeMoorJasper/parcel-plugin-svelte#15]
# 🐛 Bug Report

## 🎛 Configuration (.svelterc, package.json, cli command)

Upgraded all my projects and none of them work any more, with the error `component Xxx is not a function`

If you simply clone this project and then point parcel at the index.html file, you will notice that you get the same error:

## 💻 Code Sample

```javascript
➜  npx parcel index.html
Server running at http://localhost:1234 
✨  Built in 2.35s.
```

In your browser console:

```
component.svelte:22 Uncaught TypeError: (0 , _component2.default) is not a function
    at Object.parcelRequire.4../components/app/component.svelte (component.svelte:22)
    at newRequire (main.5ce53338.js:48)
    at parcelRequire.12 (main.5ce53338.js:75)
    at main.5ce53338.js:80
parcelRequire.4../components/app/component.svelte @ component.svelte:22
newRequire @ main.5ce53338.js:48
parcelRequire.12 @ main.5ce53338.js:75
(anonymous) @ main.5ce53338.js:80
```

## 🌍 Your Environment

This project's exact dependencies, and NodeJS 9.10.1

**Comment by antony** [comment]
@DeMoorJasper doing so as we speak. I think I replicated the problem incorrectly so I'm looking at ways to replicate it properly first.

**Comment by antony** [comment]
Ok, in trying to recreate this I encountered a different blocking bug which means that I can't progress, so...

https://github.com/antony/parcel-plugin-svelte-bug

It looks like parcel is telling me that my `component.svelte` can't be resolved. However, if you remove the `parcel-plugin-svelte` package using `npm remove --save-dev parcel-plugin-svelte`, the project compiles again, but obviously the component doesn't work.

Slightly confused as to what's going on here.

**Comment by DeMoorJasper** [comment]
@antony I can probably write an adapter for that I’ll reopen untill I’ve had some time to figure this one out

<sub>Sent with <a href="http://githawk.com">GitHawk</a></sub>

**Comment by antony** [comment]
Ok, I have to reopen this as I'm still encountering it even after switching to `parcel-bundler`. Sorry to be a pain, it seems to be exceptionally hard to track this down.

Here's a working minimal example as a branch of my seed project:

https://github.com/antony/svelte-seed/tree/bug/plugin-svelte

**Comment by antony** [comment]
Ok I've tracked it down. It feels like this might be a parcel bug or a svelte bug, but I can't replicate it on the svelte repl...

https://github.com/antony/parcel-plugin-svelte-bug

If you use a class name in the 'style' tag of a component which contains a '-', it refuses to compile the component.

In the example above, if you change:

```html
<style>
  .the-time {
    font-weight: bold;
  }
</style>
```

to
```html
<style>
  .theTime {
    font-weight: bold;
  }
</style>
```

It compiles the component without issue.

Really weird!

**Comment by DeMoorJasper** [comment]
Awesome, wondering if this goes up all the way to parcel. It’s probably not really related to the plugin, I’ll def look further into this and open a pr/issue in the project this originates from

<sub>Sent with <a href="http://githawk.com">GitHawk</a></sub>

**Comment by DeMoorJasper** [comment]
that's strange it works for vue, I'll have a look at it soonish

**Comment by DeMoorJasper** [comment]
Should be fixed in 1.1.4, this is however a hotfix and I'll create a bug report on the parcel repo and follow it up there.

**Comment by antony** [comment]
@DeMoorJasper This is excellent - it works brilliantly - you are an absolute legend!
